tdrs-backend/reports/zap.conf

```diff
@@ -121,4 +121,4 @@
 100000	WARN	(A Client Error response code was returned by the server)
 100001	WARN	(Unexpected Content-Type was returned)
 100001	OUTOFSCOPE	http://web:8080/v1/login
-100001	OUTOFSCOPE	https://tdp-backend-staging.acf.hhs.gov/v1/login
\ No newline at end of file
+100001	OUTOFSCOPE	https://tdp-backend-staging.acf.hhs.gov/v1/login
```

Comment:
@raftmsohani do we need to add this ignore to prod url as well?

Reply:
@raftmsohani additionally, ive noticed that the outofscope tag doesnt work consistently. have we tried `ignore`?

Reply:
> @raftmsohani additionally, ive noticed that the outofscope tag doesnt work consistently. have we tried `ignore`?

Haven't tested outofscope tag, can review and get back to you

Reply:
That's just artifact of linting. No changes on the URL